xarray/tests/test_dask.py

```diff
@@ -491,6 +491,7 @@ def test_dot(self):
         lazy = self.lazy_array.dot(self.lazy_array[0])
         self.assertLazyAndAllClose(eager, lazy)
 
+    @pytest.mark.skipif(LooseVersion(dask.__version__) < "2.0", reason="needs meta")
```

Comment:
if anybody changes ``inline_dask_repr`` to expect _meta to always be there, dask < 2.0 will start crashing and no test will notice.

Reply:
added a test for this, though I'm if we have a CI configuration currently that tests old versions of dask (that would be good to add)